.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature request
```

Comment:
Does this need to be quoted or something?

Reply:
did a bit of digging on another repo - looks like it's a comma separated list, so no.